core/app/Models/Client.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Laravel\Passport\Client as PassportClient;
+
+class Client extends PassportClient
+{
+    use HasFactory;
+
+    //================================================ relationships =========================================================
+
+    /**
+     * Get the valid domains for the client.
+     */
+    public function valid_domains()
+    {
+        return $this->hasMany('App\Models\ValidDomain');
```

Comment:
use class property instead of alias string.
for example in this case use the  ValidDomain in top of file by `use App\Models\ValidDomain` (in this case ValidDomain and Client class are in same dir so you don't need use
and change this line to `return $this->hasMany(ValidDomain::class)` because in last laravel updates, this kind of addressing is deprecating. for example in laravel 8,  string addressing to controllers in routes is deprecated